packages/off-chain/outputs/metadata.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
When Marvin has finished the composeDB query, I think we might want to pass it into the bacalhau container (is part of the contributions data input) and return it as output in this file, so that anyone can verify the contributions input param (verify that the contributions param is in fact the result of the composeDB query).

Reply:
I think, we can create a new file, which store the input. or we can just store the hash of all inputs.
But as the input data is on IPFS, I think returning the whole data is redundant.